1.  Select a dataset (at least one).
    - Wine Quality Dataset  
2. Identify suitable samples within the dataset that allow you to test a meaningful hypothesis using a t-test.
    - Nullhypothese: Es gibt keinen Unterschied im Durchschnitt der flüchtigen Säure zwischen Rot- und Weißweinen  
    - Alternativhypothese: Es gibt einen signifikanten Unterschied
3. Decide which type of t-test is appropriate (e.g., one-sample, independent two-sample, or paired).
Implement the t-test in Python.
-  Unabhängiger Zweistichproben-t-Test: da es um zwei unterschiedliche Gruppen handelt


In [9]:
import pandas as pd
from scipy import stats

red = pd.read_csv('winequality-red.csv', sep=';')
white = pd.read_csv('winequality-white.csv', sep=';')

group_red = red['volatile acidity']
group_white = white['volatile acidity']

#t-test
t_stat, p_value = stats.ttest_ind(group_red, group_white, equal_var=False)

print(f"Mittelwert Rotwein:  {group_red.mean():.4f}")
print(f"Mittelwert Weißwein: {group_white.mean():.4f}")
print(f"t-statistic: {t_stat:.4f}")
print(f"p-value:     {p_value}") 

if p_value < 0.05:
    print("H0 ablehnen")
else:
    print("H0 beibehalten")

Mittelwert Rotwein:  0.5278
Mittelwert Weißwein: 0.2782
t-statistic: 53.0591
p-value:     0.0
H0 ablehnen


4. Interpret and explain the results clearly, linking them back to your hypothesis:

Der p-Wert liegt weit unter dem Signifikanzniveau von 0.05, was bedeutet die **Nullhypothese wird abgelehnt**  

Rotweine enthalten im Durchschnitt (0.52) signifikant mehr flüchtige Säure als Weißweine (0.28).